tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -1204,18 +1204,14 @@ cc_library(
 cc_library(
     name = "amdgpu_compiler_impl",
     srcs = if_rocm_is_configured([
```

Comment:
I refactored the BUILD file a bit to require less heavy use of `if_rocm_is_configured` here. You will need to resolve conflict, sorry, but it should be straightforward.

Reply:
thanks. will do.